### PR TITLE
update copy; remove link

### PR DESCRIPTION
**Pull request by rianfowler**
Address [this](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11760#issuecomment-409044784).

The copy edits remove the need to control the checkbox with a link button. 👍 

screen shot 2018-08-06 at 12 08 56

### CODE REVIEW

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 1]
```diff
@@ -209,14 +204,9 @@ export class SchoolSelectField extends React.Component {
...
                 <strong>
-                  {'No schools found. '}
-                </strong>
-                {'Please try entering a different search term (school name or address), or '}
-                <button
-                  className="va-button-link"
-                  onClick={this.handleManuallyEnterClicked}>
-                  {'manually enter your school’s information by clicking this link.'}
-                </button>
+                  {'We can’t find your school'}
```
Comment (erikphansen) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: should this end with a period?

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 2]
```diff
@@ -209,14 +204,9 @@ export class SchoolSelectField extends React.Component {
...
+                </strong><br/>
+                {'We’re sorry. We can’t find any school that matches your entry. Please try entering a different school name or address. Or, you can check the box to enter your school information yourself'}
```
Comment (erikphansen) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: should this end with a period?

File: src/applications/edu-benefits/components/SchoolSelectField.jsx [hunk 3]
```diff
@@ -9,8 +9,7 @@ import {
   clearSearch,
   searchInputChange,
   selectInstitution,
-  searchSchools,
-  setCannotFindSchool
+  searchSchools
```
Comment (erikphansen) [inline src/applications/edu-benefits/components/SchoolSelectField.jsx]: Does anyone else have strong feelings on enforcing dangling commas to reduce diff noise like this? I used to hate them but have recently become a convert.

**Review by erikphansen — APPROVED** [review]
Just a couple periods missing (maybe).

**Review by cehsu — APPROVED** [review]

File: src/applications/edu-benefits/components/SchoolSelectField.jsx (hunk 2 above)
Comment (rianfowler) [reply src/applications/edu-benefits/components/SchoolSelectField.jsx]: Good catch added.

File: src/applications/edu-benefits/components/SchoolSelectField.jsx (hunk 3 above)
Comment (rianfowler) [reply src/applications/edu-benefits/components/SchoolSelectField.jsx]: I don't mind either way. I wish we could enforce alphabetization.

File: src/applications/edu-benefits/components/SchoolSelectField.jsx (hunk 1 above)
Comment (rianfowler) [reply src/applications/edu-benefits/components/SchoolSelectField.jsx]: No period on this one as per the copy.

**Pull request merged**